lib/core-classes/DeviceService.ts

```diff
@@ -406,6 +406,63 @@ export class DeviceService {
     return results;
   }
 
+  async importCatalog (
+    catalog: JSONObject[],
+    { strict, options }: { strict?: boolean, options?: JSONObject }) {
+      const results = {
+        errors: [],
+        successes: [],
+      };
+
+      const pluginConfigDocument = await this.sdk.document.get(
+        this.config.adminIndex,
+        this.config.configCollection,
+        'plugin--device-manager');
+
+      const catalogStrategy: boolean = pluginConfigDocument._source['device-manager'].provisioningStrategy === 'catalog';
+
+      if (! catalogStrategy) {
```

Comment:
It could be useful to allows to import catalog even if the strategy is not set. Users may be able to prepare their catalog this way for example

Reply:
I agree